plugins/lsp/language-servers/nil-ls-settings.nix

```
@@ -0,0 +1,58 @@
+{ lib, helpers }:
+# Options:
+#  - https://github.com/oxalica/nil/blob/main/docs/configuration.md?plain=1#
+with lib;
+{
+  formatting = {
+    command = helpers.defaultNullOpts.mkNullable (types.listOf types.str) "null" ''
+      External formatter command (with arguments).
+      It should accepts file content in stdin and print the formatted code into stdout.
+    '';
+  };
+  diagnostics = {
```

```suggestion
  };

  diagnostics = {
```

Please, add a blank line between each option declaration.